website/thaliawebsite/settings.py

```diff
@@ -396,6 +392,10 @@ def show_toolbar(request):
         "task": "members.tasks.expiration_announcement",
         "schedule": crontab(minute=0, hour=6, day_of_month=8, month_of_year=8),
     },
+    "studylongexpirationwarning": {
+        "task": "members.tasks.expiration_warning",
```

Comment:
I'm thinking it would be good to send a second reminder sometime in september, _before- minimization of the users who accidentally ignored the first email

Reply:
Shouldn't that then be a different, maybe a bit more urgent mail

Reply:
Yup